source/server/filter_chain_manager_impl.h

```diff
@@ -197,12 +197,13 @@ class FilterChainManagerImpl : public Network::FilterChainManager,
   using FcContextMap =
       absl::flat_hash_map<envoy::config::listener::v3::FilterChain,
                           Network::DrainableFilterChainSharedPtr, MessageUtil, MessageUtil>;
-  FilterChainManagerImpl(const Network::Address::InstanceConstSharedPtr& address,
+  FilterChainManagerImpl(absl::string_view listener_name,
```

Comment:
To match dual stack, will Network::Address better than the listener_name for filterChainMatch? For each different address in the listener, the filterChain for each address should be different?

Reply:
This isn't used for filterChainMatch. Each ListenerImpl will have a FilterChainManager, and the original `address` is just used for logging from the code. But now, each ListenerImpl has multiple addresses, so I use the name instead of it.